rosa_tutorials/rosa-mobb-deploy-api-data-protection.adoc

```diff
@@ -0,0 +1,591 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-deploy-api-data-protection"]
+= Tutorial: Deploying OpenShift API for Data Protection on a ROSA cluster
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-deploy-api-data-protection
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2022-09-14'
+//title: Deploying OpenShift API for Data Protection on a ROSA cluster
+//tags: ["ROSA", "AWS", "STS", "OADP", "Velero", "Backup", "Restore", "Storage"]
+//authors:
+//  - Paul Czarkowski
+//  - Dustin Scott
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* A ROSA classic cluster
+// The prereq above is originally linked to a mobb doc (which in turn also points to an OCP doc about the same topic). Not clear whether this prereq should be linked to either of those pages in this PR.
+
+.Environment
+
+* Prepare the environment variables:
++
+[NOTE]
+====
+Change the cluster name to match your ROSA cluster and ensure you're logged into the cluster as an Administrator.
+Ensure all fields are outputted correctly before moving on.
+====
++
+[source,terminal]
+----
+$ export CLUSTER_NAME=my-cluster
+$ export ROSA_CLUSTER_ID=$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .id)
+$ export REGION=$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .region.id)
+$ export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed  's|^https://||')
+$ export AWS_ACCOUNT_ID=`aws sts get-caller-identity --query Account --output text`
+$ export CLUSTER_VERSION=`rosa describe cluster -c ${CLUSTER_NAME} -o json | jq -r .version.raw_id | cut -f -2 -d '.'`
+$ export ROLE_NAME="${CLUSTER_NAME}-openshift-oadp-aws-cloud-credentials"
+$ export AWS_PAGER=""
+$ export SCRATCH="/tmp/${CLUSTER_NAME}/oadp"
+$ mkdir -p ${SCRATCH}
+$ echo "Cluster ID: ${ROSA_CLUSTER_ID}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+
+== Prepare AWS Account
+
+. Create an IAM Policy to allow for S3 Access:
++
+[source,terminal]
+----
+$ POLICY_ARN=$(aws iam list-policies --query "Policies[?PolicyName=='RosaOadpVer1'].{ARN:Arn}" --output text)
+if [[ -z "${POLICY_ARN}" ]]; then
+cat << EOF > ${SCRATCH}/policy.json
```

Comment:
Add a $ before `cat` - it's a sneaky one

$ cat << EOF > ${SCRATCH}/policy.json